transform.go

```diff
@@ -446,6 +446,8 @@ func (t *Transformer) makeRhsParam(expr Expression) interface{} {
 			return "??ERROR??"
 		}
 		return NewFastVal(regex)
+	} else if expr == nil {
+		return NewFastVal(nil)
```

Comment:
I think this no longer makes sense with the latest changes, I believe in the new code this is already implied.

Reply:
done